assets/js/base/components/cart-checkout/totals/coupon/index.tsx

```diff
@@ -63,30 +68,33 @@ export const TotalsCoupon = ( {
 		if ( currentIsLoading.current !== isLoading ) {
 			if ( ! isLoading && couponValue && ! validationError ) {
 				setCouponValue( '' );
+				setCouponFormClass( 'screen-reader-text' );
 			}
 			currentIsLoading.current = isLoading;
 		}
 	}, [ isLoading, couponValue, validationError ] );
 
 	return (
-		<Panel
-			className="wc-block-components-totals-coupon"
-			hasBorder={ false }
-			initialOpen={ initialOpen }
-			title={
-				<Label
-					label={ __(
-						'Coupon code',
+		<>
+			{ couponFormClass === 'screen-reader-text' && (
+				<a
+					role="button"
+					href="#wc-block-components-totals-coupon__form"
```

Comment:
This href needs an element that has the id `wc-block-components-totals-coupon__form`, and I noticed the form has this as a class, but not as an id.

Reply:
I have added the ID to the element. 👍